regulations/static/regulations/js/source/views/comment-review-view.js

```diff
@@ -0,0 +1,81 @@
+'use strict';
+
+var $ = require('jquery');
+var _ = require('underscore');
+var Backbone = require('backbone');
+Backbone.$ = $;
+
+var CommentView = require('./comment-view');
+
+var CommentReviewView = Backbone.View.extend({
+  events: {
+    'click #submit': 'submit'
+  },
+
+  initialize: function(options) {
+    Backbone.View.prototype.setElement.call(this, '#' + options.id);
+    this.template = $('#comment-template').html();
+    this.$content = this.$el.find('#comment');
+    this.docNumber = this.$el.data('doc-number');
+    this.prefix = 'comment:' + this.docNumber;
+
+    this.showComments();
+  },
+
+  render: function() {},
+
+  showComments: function() {
+    this.comments = _.chain(_.keys(window.localStorage))
+      .filter(function(key) {
+        return key.indexOf(this.prefix) === 0;
```

Comment:
That's an option. The only issue that comes to mind is that `localStorage` isn't nested--it maps string keys to string values. So it would be more like

`JSON.parse(window.localStorage.getItem('commentEditor'))[this.docNumber].comments`

Using a single blob for all N&C-related data is also potentially tricky because we don't get atomic writes--we have to overwrite the entire namespace every time we change anything. We can definitely do that, but we'd have to be careful that one write doesn't blow away another.